src/sentry/static/sentry/app/components/assigneeSelector.jsx

```diff
@@ -22,6 +22,7 @@ const AssigneeSelector = React.createClass({
 
   mixins: [
     Reflux.listenTo(GroupStore, 'onGroupChange'),
+    Reflux.connect(MemberListStore, 'memberList'),
```

Comment:
tbh it seems like what we're doing with init() and the component's getInitialState seems more straightforward to me. do you agree?